1_fetch/src/my_happy_script.R

```diff
@@ -0,0 +1,96 @@
+
+library(dplyr)
```

Comment:
This script is now outdated, isn't it? You can delete it.

Reply:
Thanks, and done.